src/core/net/ip6_address.hpp

```diff
@@ -313,6 +313,26 @@ class Address : public otIp6Address
         SetPrefix(aMeshLocalPrefix.m8, Mle::MeshLocalPrefix::kLength);
     }
 
+    /**
+     * This method sets the prefix content of the Prefix-Based Multicast Address.
+     *
+     * @param[in]  aPrefix         A buffer containing the prefix.
+     * @param[in]  aPrefixSize     The prefix size (in bytes).
```

Comment:
The more general form is to have prefix length in bits. Thoughts?

Reply:
Updated. Thanks.